models/uniswap/arbitrum/uniswap_arbitrum_trades_legacy.sql

```diff
@@ -6,7 +6,7 @@
 }}
 
 {% set uniswap_models = [
-'uniswap_v3_arbitrum_trades'
+'uniswap_v3_arbitrum_trades_legacy'
```

Comment:
i noticed elsewhere that this setup of jinja code made us miss applying `_legacy` in refs downstream

no big deal -- just pointing it out and we'll need to monitor on migrations moving forward

Reply:
good catch!